src/getting-started.md

```diff
@@ -377,10 +377,21 @@ team member you've been working with), you can specifically request them by
 writing `r? @user` (e.g. `r? @eddyb`) in either the original post or a followup
 comment.
 
+Please note that the reviewers are humans, who for the most part work on `rustc`
+in their free time. This means that they can take some time to respond and review
+your PR. It also means that reviewers can miss some PRs that are assigned to them.
+
+To try to move PRs forward, the Triage WG regularely goes through all PRs that
+are waiting for review and haven't been discussed for at least a week. If you
+don't get a review within around 10 days, feel free to ask the Triage WG on
```

Comment:
15 days is our current policy or you can say "2 weeks"

Reply:
Changed in 650a5b8, if you think that's correct we can merge this